solr/server/etc/jetty.xml

```diff
@@ -126,6 +126,15 @@
           </New>
         </Arg>
       </Call>
+      <Call name="addRule">
```

Comment:
shouldn't this go in the `jetty-https.xml` file? Otherwise the server is going to tell the browser to expect https requests for an http server.

Reply:
Ok I see now that this isn't an issue when using HTTP only. Ignore

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security#description